src/components/slider/slider.tsx

```diff
@@ -189,7 +189,8 @@ export class Slider implements LabelableComponent, FormComponent {
     const rightThumbOffset = `${mirror ? maxInterval : 100 - maxInterval}%`;
 
     const handle = (
-      <button
+      <div
```

Comment:
Can you share the reasoning behind the `button` to `div` change? As a best practice, we want to have our components use as much semantic HTML as possible.

Reply:
The reason for choosing div other than button is the role compatibility. Assigning slider role to button is not allowed but w3 docs doesn't explain the limitations of `role=slider`.

Reply:
Gotcha. Thanks for the breakdown.